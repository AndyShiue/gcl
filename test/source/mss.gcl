con N : Bool
con A : array (0 .. N] of Int
