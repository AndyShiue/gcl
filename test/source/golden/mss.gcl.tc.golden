Error Type Error ./test/source/mss.gcl:4:2-7
The definition Falsee is not in scope